concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -1042,10 +1007,17 @@ jobs:
                   --name '' \
                   blank-runtime-config.yml
 
+                cp bosh-runtime-config-vars-store/bosh-runtime-config-vars-store.yml \
+                   new-bosh-runtime-config-vars-store/bosh-runtime-config-vars-store.yml
+
                 bosh update-runtime-config \
```

Comment:
I thik it is good to still do two steps calling `bosh interpolate` and then `bosh update-runtime-config`.

The reason is that in `bosh interpolate` you can pass this two handy parameters:

```
          --var-errs             Expect all variables to be found, otherwise error
          --var-errs-unused      Expect all variables to be used, otherwise error
```

Reply:
Thanks, @keymon - I *knew* that `update-runtime-config` doesn't accept `--vars-errs` and that was why I'd broken it into 2 steps (or, at least, 2 commands), but I didn't flag it up in review yesterday.  Good call.

Reply:
@keymon Addressed in the recent push.